build-system/test-configs/forbidden-terms.js

```diff
@@ -461,6 +461,7 @@ const forbiddenTermsGlobal = {
     message: requiresReviewPrivacy,
     allowlist: [
       'extensions/amp-access/0.1/amp-access-iframe.js',
+      'extensions/amp-ad-network-mgid-impl/0.1/amp-ad-network-mgid-impl.js',
```

Comment:
Using storage would require a privacy review, since you would be storing information against the publisher domain. Ads has not been cleared to do that yet. I would recommend finding a different way to do it. (you could iframe and do whatever you want on a 3p iframe)

Reply:
I thought we can't use 3p iframe in amp-story-auto-ads. Is there any example of implementing this?

Reply:
You are correct in that story ads should be same domain iframe. Is it possible to find a solution without using localStorage? I would say its unlikely to get approved.

Reply:
For now, we use localStorage to save user identifier and pass it to our ad server on subsequent requests. We could use cookies for this purpose, but as I understand, there will be the same issue as with localStorage. Can you recommend any acceptable way of sharing our user identifier between multiple requests to ad server? 

However, if there are no acceptable way to store user identifier, we will remove that part of code.

Also, we store some data in sessionStorage. Is it allowed?

Reply:
we offer a managed storage solution that considers the many contexts amp pages are viewed (pub domain, cache domain, viewer, etc) https://github.com/ampproject/amphtml/blob/main/docs/spec/amp-var-substitutions.md#client-id maybe that can satisfy your use case?

Reply:
Ok. I've removed usage of localStorage and sessionStorage. Is it ok now?

Reply:
@calebcordry Can you confirm it's ok now, please?

Reply:
Sorry lost track, feel free to ping again if I dont respond for a day or two.